src/shared/config-db.ts

```diff
@@ -183,60 +85,160 @@ export const DatabaseConfigScheme = {
   client: {
     doc: 'Which database client should we use',
     format: ['mysql', 'sqlite3'],
-    default: 'sqlite'
+    default: 'sqlite',
+    env: 'DB_CLIENT'
   },
   version: {
     doc: 'What database version should we use',
     format: String,
-    default: '5.5'
+    default: '5.5',
+    env: 'DB_VERSION'
   },
   useNullAsDefault: {
     doc: 'whether or not to use null for everything not specified',
-    format: 'Boolean',
-    default: false
+    format: Boolean,
+    default: false,
+    env: 'DB_USE_NULL_AS_DEFAULT'
   },
   connection: {
-    doc: 'Connection object specifying properties like host, port, user etc.',
-    format: DbConnectionFormat.name,
-    default: null
+    filename: {
+      doc: 'Database filename',
+      format: String,
+      default: null,
+      env: 'DB_CONNECTION_FILENAME',
+      nullable: true
+    },
+    host: {
+      doc: 'Database host',
+      format: String,
+      default: null,
+      env: 'DB_CONNECTION_HOST',
+      nullable: true
+    },
+    port: {
+      doc: 'Database port',
+      format: Number,
+      default: null,
+      env: 'DB_CONNECTION_PORT',
+      nullable: true
+    },
+    user: {
+      doc: 'Database user',
+      format: String,
+      default: null,
+      env: 'DB_CONNECTION_USER',
+      nullable: true
+    },
+    password: {
+      doc: 'Database password',
+      format: String,
+      default: null,
+      env: 'DB_CONNECTION_PASSWORD',
+      nullable: true
+    },
+    database: {
+      doc: 'Database name',
+      format: String,
+      default: null,
+      env: 'DB_CONNECTION_NAME',
+      nullable: true
+    },
+    timezone: {
+      doc: 'Database timezone',
+      format: String,
+      default: null,
+      env: 'DB_CONNECTION_TIMEZONE',
+      nullable: true
+    }
   },
   pool: {
-    doc: 'Pool object specifying tarn pool properties',
-    format: DbPoolFormat.name,
-    default: null
+    min: {
+      doc: 'Database timezone',
```

Comment:
Doc discriptions look like copy-pasta?

I think they all need to be updated to match the config pool paramater's description.